packages/trpc/server/routers/viewer/availability.tsx

```diff
@@ -93,6 +95,23 @@ export const availabilityRouter = createProtectedRouter()
       };
     },
   })
+  .query("user", {
+    input: z.object({
+      user: z.string(),
```

Comment:
```suggestion
      username: z.string(),
```

To match the comment above